src/components/shared-tabs.scss

```
@@ -0,0 +1,137 @@
+// shared-tabs.scss — visual treatment shared between ChoiceTabs and
+// PlatformTabs. The two components stay locked in sync by reading the same
+// --tab-card-* custom properties below. If you want to tweak the look,
+// tweak the variables. The rspress --home-features-item-bg resolves to
+// rgba(255,255,255,0.5) in light mode (invisible on the white page), so
+// we use currentColor mixes that trace the outline in BOTH themes.
+
+:root {
+  --tab-card-bg: color-mix(in srgb, currentColor 5%, transparent);
```

Stylelint (17.12.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)